README.md

```diff
@@ -176,7 +176,7 @@ The Azure Key Vault Provider offers four modes for accessing a Key Vault instanc
 
 **OPTION 1 - Service Principal**
 
-> Supported with linux and windows
+> Supported with Linux and Windows
```

Comment:
Looking at this more, I wonder if we should move all the details for these options in its own readme but keep the top level here are different ways to auth in this readme. wdyt?

Reply:
Yeah, that makes sense. I'll update that!